sql/tablewriter.go

```diff
@@ -271,6 +271,9 @@ func (tu *tableUpserter) init(txn *client.Txn) error {
 		if err != nil {
 			return err
 		}
+		// The fetchCols can also contain columns undergoing mutation; update
+		// requestCols.
+		requestedCols = tu.ru.fetchCols
```

Comment:
to match the rest of these, leave requestedCols alone here and iterate tu.ru.fetchCols below instead